pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: Inclusive Language style guide
+Description: This style guide outlines the guideline for using inclusive languages in documentation.
+---
+
+# Inclusive Language
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+## 1. Culturally inclusive language
+
+Some phrases and words are popularly used only in specific regions of the world, thus our audience from other parts 
+of the world might need help to relate to the same. Hence, we should try to avoid using such phrases in our documentation. 
+Some examples of Region Specific Language are -
+- “It’s not rocket science” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+- Phrases written in regional languages instead of English.
+- “It’s not a piece of cake” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+
+## 2. Gender neutral language 
```

Comment:
One could add to this point "avoid the use of stereotypes": such as linking men with strength, women with care work, male homosexuality with sensitivity, etc.

Reply:
added the changes!